static/js/hotkey.js

```diff
@@ -571,6 +571,52 @@ exports.process_hotkey = function (e, hotkey) {
         return false;
     }
 
+    if (event_name === "next_unread_topic") {
+        // message_header of selected_message
+        var message_header = $(".selected_message").closest(".recipient_row").find(".message_header");
+        var selected_stream = ($(message_header).find(".stream_label").html()).replace(/ |\n/g,'');
+        var selected_topic = ($(message_header).find(".stream_topic a").text()).replace(/ |\n/g,'');
```

Comment:
These you don't see to parse from the DOM using jquery; use `narrow.operators()` or `narrow.filter().operands('stream')` to get these data from the declarative data structures for which narrow the user is in.